framework/meta-lib/src/contract/generate_snippets/snippet_template_gen.rs

```diff
@@ -230,12 +219,49 @@ pub(crate) fn write_config_struct_impl(file: &mut File) {
         match self.chain_type {{
             ChainType::Real => false,
             ChainType::Simulator => true,
+        }}
     }}
-    }}
-    }}
-"#
+}}"#
     )
     .unwrap();
+}
 
-    write_newline(file);
+pub(crate) fn write_chain_sim_test_to_file(file: &mut File, crate_name: &str) {
+    writeln!(
+        file,
+        r#"use multiversx_sc_snippets::imports::*;
+use rust_interact::ContractInteract;
+
+// Simple deploy test that runs using the chain simulator configuration.
+// In order for this test to work, make sure that the `config.toml` file contains the chain simulator config (or choose it manually)
+// The chain simulator should already be installed and running before attempting to run this test.
+// The chain-simulator-tests feature should be present in Cargo.toml.
+// Can be run with `cargo test --features chain-simulator-tests`.
```

Comment:
we have this command in `sc-meta`. Better replace `cargo test --features chain-simulator-tests` with `sc-meta test -c`